doc/source/whatsnew/v0.24.0.txt

```diff
@@ -368,6 +368,37 @@ is the case with :attr:`Period.end_time`, for example
 
    p.end_time
 
+.. _whatsnew_0240.api_breaking.frame_to_dict_index_orient:
+
+Raise ValueError in ``DataFrame.to_dict(orient='index')``
```

Comment:
A dedicated section like this is typically reserved for larger changes, which I don't think applies here. Can you instead just add this as a one-line note in `Other API Changes`?

Reply:
My mistake I didn't see that comment; with that being the case go ahead and leave as is